### ISSUE TITLE
Remember tubclean count (feature request)

### ISSUE THREAD

**Issue by towerhand** [ollo69/ha-smartthinq-washer#11]
This is just a small tweak, the tubclean count shows N/A after turning off the washer for a while. I think it would make more sense to maintain the counter and update it as the number changes.

Thanks for all the updates, "patiently" waiting for the dryer release that you are working on.

**Comment by ollo69** [comment]
I just released Dryer, test can start now.Please provide feed-back and send to me logs (possible debug), because not having Dryer I wasnìt able to do any test
It is strange that tubclean come back to N/A. This should happen only after HA reboot.

**Comment by towerhand** [comment]
Ok, I'll create a new issue for the dryer testing.

**Comment by towerhand** [comment]
Is there any way for the counter to survive a restart?

**Comment by ollo69** [comment]
Problem is that at restart, when washer is off, this value is not available. As I know in HA is not possible to save a state and restore at reboot.

**Comment by towerhand** [comment]
Ok thanks, I'll close this.
